crates/rpc/src/handlers/validator.rs

```diff
@@ -7,22 +7,28 @@ use actix_web::{
 use ream_beacon_api_types::{
     error::ApiError,
     id::{ID, ValidatorID},
-    query::{IdQuery, StatusQuery},
+    query::{AttestationQuery, IdQuery, StatusQuery},
     request::ValidatorsPostRequest,
-    responses::BeaconResponse,
+    responses::{BeaconResponse, DataResponse},
     validator::{ValidatorBalance, ValidatorData, ValidatorStatus},
 };
 use ream_bls::PublicKey;
 use ream_consensus::{
-    constants::SLOTS_PER_EPOCH, electra::beacon_state::BeaconState, validator::Validator,
+    attestation_data::AttestationData, constants::SLOTS_PER_EPOCH,
+    electra::beacon_state::BeaconState, validator::Validator,
 };
-use ream_storage::db::ReamDB;
+use ream_fork_choice::store::Store;
+use ream_operation_pool::OperationPool;
+use ream_storage::{db::ReamDB, tables::Field};
 use serde::Serialize;
 
 use super::state::get_state_from_id;
 
 const MAX_VALIDATOR_COUNT: usize = 100;
 
+//  For slots in Electra and later, this AttestationData must have a committee_index of 0.
+const ELECTRA_COMMITTEE_INDEX: u64 = 0;
```

Comment:
```suggestion
///  For slots in Electra and later, this AttestationData must have a committee_index of 0.
const ELECTRA_COMMITTEE_INDEX: u64 = 0;
const MAX_VALIDATOR_COUNT: usize = 100;
```

the comment should be a doc comment and sorted

Reply:
Can we include rules for these in clippy ? I have made changes though